rpc/README.md

```diff
@@ -302,19 +302,38 @@ curl -d '{"id": 2, "jsonrpc": "2.0", "method":"local_node_info","params": []}' -
 {
     "jsonrpc": "2.0",
     "result": {
-        "addresses": [
-            {
-                "address": "/ip4/0.0.0.0/tcp/12344/p2p/QmWRU2NSro4wKgVbFX6y8SPFkcJ1tE2X5xzk9msMhdRmdS",
-                "score": 1
-            }
-        ],
+        "address": "/ip4/0.0.0.0/tcp/12344/p2p/QmWRU2NSro4wKgVbFX6y8SPFkcJ1tE2X5xzk9msMhdRmdS",
```

Comment:
Why change this?

Reply:
ops, my mistake, some copy/paste and rebase error... will resolve it later, thanks.

Reply:
@jjyr resolved, please review again, and for the new `get_peers` rpc, need your help to fix this issue:
https://github.com/nervosnetwork/ckb/blob/63fa068566c58997ad37c2edd46300c41d23c40a/rpc/src/module/net.rs#L49

We can submit a new fix after this PR and Identify refactoring PR are merged.